tieredstore/src/main/java/org/apache/rocketmq/tieredstore/provider/TieredFileSegment.java

```diff
@@ -429,15 +431,29 @@ public static class TieredFileSegmentInputStream extends InputStream {
 
         private final FileSegmentType fileType;
         private final List<ByteBuffer> uploadBufferList;
```

Comment:
This stream is more complicated now, should we separate it into a new file?

Reply:
> This stream is more complicated now, should we separate it into a new file?

Yep~